anchors/anchor.go

```diff
@@ -133,7 +133,7 @@ func NewCommitData(blockHeight uint64, anchorID AnchorID, documentRoot DocumentR
 }
 
 // GenerateCommitHash generates Keccak256 message from AnchorID, CentID, DocumentRoot
-func GenerateCommitHash(anchorID AnchorID, centrifugeID identity.CentID, documentRoot DocumentRoot) []byte {
+func GenerateCommitHash(anchorID AnchorID, centrifugeID identity.DID, documentRoot DocumentRoot) []byte {
```

Comment:
After the migration, we should think about if need the identity address as part of the anchor commit.
In case we using the execute method for anchoring later.